wbooks/src/redux/book/reducer.ts

```diff
@@ -1,25 +1,16 @@
-import { AnyAction } from 'redux';
+import { completeReducer, completeState, createReducer } from 'redux-recompose';
 import { BookState } from '@interfaces/bookState';
 
 import { actions } from './actions';
 
-const initialState: BookState = {
-  books: [],
-  error: '',
-  loading: false
+const initialState: BookState = completeState({
+  books: []
+});
```

Comment:
Wrong, check again the [docs](https://github.com/Wolox/redux-recompose/tree/master/src/completers/completeState)

Reply:
I'm now wrapping it with `description: { ... }`